.github/ISSUE_TEMPLATE/black.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Filename should `black.md` or `blank.md`? Maybe a typo

Reply:
ops, thanks!!